lib/main_containers.pm

```diff
@@ -176,6 +176,7 @@ sub load_host_tests_docker {
         loadtest 'containers/buildx';
         loadtest 'containers/rootless_docker';
     }
+    loadtest('containers/skopeo', run_args => $run_args, name => $run_args->{runtime} . "_skopeo");
```

Comment:
```suggestion
    loadtest('containers/skopeo', run_args => $run_args);
```
I don't think that we ever need to distinguish the `skopeo` test by runtime or am I missing something? :thinking:

Reply:
> I don't think that we ever need to distinguish the `skopeo` test by runtime or am I missing something? 🤔

In the ticket there's a reference to customers who do this.  Whether it's supported or not, I don't know.

Reply:
roger that! Reverted back to original `loadtest('containers/skopeo', run_args => $run_args, name => $run_args->{runtime} . "_skopeo");`